### PR TITLE
Fix German translation No.7

### PR DESCRIPTION
**Pull request by erazortt**
Fix the German translation, Part 7 of 8
See also https://github.com/ihhub/fheroes2/issues/7860
And https://github.com/ihhub/fheroes2/pull/7865

### CODE REVIEW

File: files/lang/de.po [hunk 1]
```diff
@@ -9964,10 +9960,10 @@ msgid ""
 "doesn't shrink. You realize to your delight that the wood is enchanted, so "
 "you take it along."
 msgstr ""
-"Ihr macht eine Pause, um euch auszuruhen und ein Kochfeuer anzuzünden, und "
-"zieht Holz aus einem nahe gelegenen Haufen toten Holzes. Als ihr weiter Holz "
-"aus dem Stapel zieht, bemerkt ihr, dass es nicht schrumpft. Zu eurer Freude "
-"stellt ihr fest, dass das Holz verzaubert ist, also nehmt ihr es mit."
+"Neben einem Holzestoß macht Ihr eine Rast, um euch auszuruhen. Für ein "
+"Lagerfeuer zeiht Ihr einige Holzscheite aus dem Haufen. Dabei bemerkt Ihr, "
```
Comment (Laserlicht) [inline files/lang/de.po:9964]:
```suggestion
"Lagerfeuer zieht Ihr einige Holzscheite aus dem Haufen. Dabei bemerkt Ihr, "
```

File: files/lang/de.po [hunk 2]
```diff
@@ -10296,23 +10283,22 @@ msgid ""
 "answers: \"You think this is funny? Fine. You can carry it. I much prefer "
 "flying anyway.\""
 msgstr ""
-"Als Sie einen kleinen Hügel keimen, kommen Sie auf einen lächerlichen Blick. "
-"Eine Fee versucht es, ein Zepter zu tragen, das fast so groß ist, wie sie "
-"selbst. Sie versuchen nicht zu lachen und fragen: \"Hilfe gebraucht?\" Die "
-"Fee gleicht auf Sie und antwortet: \"Du findest das lustig? Na gut. Du "
-"kannst es tragen. Ich fliege sowieso viel lieber.\""
+"Auf einem Huegel bietet sich Euch ein lächerlicher Anblick. Eine Fee "
```
Comment (Laserlicht) [inline files/lang/de.po:10286]:
```suggestion
"Auf einem Hügel bietet sich Euch ein lächerlicher Anblick. Eine Fee "
```

**Review by Laserlicht — CHANGES_REQUESTED** [review]

File: files/lang/de.po [hunk 3]
```diff
@@ -9703,59 +9702,58 @@ msgid ""
...
-"Anhänger die Untoten unter eurer Kontrolle vor den Zaubern des heiligen "
+"In einem schnellen und tödlichen Kampf mit einem Nekromanten gewinnt Ihr "
+"seinen magischen Anhänger. Später erfahrt Ihr von einem Zauberer, dass der "
+"Anhänger die Untoten in Euren Truppen vor den Zaubern des heiligen "
 "Wortes schützt."
 
 msgid "Wand of Negation"
 msgstr "Stab der Aufhebung"
 
 #, fuzzy
```
Comment (ihhub) [inline files/lang/de.po:9745]: Please remove this line.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @erazortt , I left one comment here. Could you please check it when you have time?

File: files/lang/de.po (hunk 3 above)
Comment (erazortt) [reply files/lang/de.po]: done

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@erazortt , thank you so much for the update!

**Pull request merged**